### ISSUE TITLE
Tutorials are jumpy and need to be smoothened

### ISSUE THREAD

**Issue by tomasstark** [laterpay/revenue-generator#252]
# Acceptance Criteria

- The tutorials should not jump around the page as you progress but should instead behave smoothly as is shown in the original prototype.
- There should be a transition added as tooltips are displayed and elements are highlighted as per the prototype.
- When tutorial scrolls, it should scroll smoothly to a new position instead of just changing scroll position which creates an unwanted jumpy effect.
- Update font size of the tutorial text (~15px as seen in the prototype).
- Make **NEXT >** links bold.

## Prototype
Original prototype can be found [here](https://www.figma.com/proto/3INU28Yy8nRuCjnBiLtxsf/0122020_WPP?node-id=7%3A5524&scaling=scale-down-width).

**Comment by ksmoore125** [comment]
@milindmore22 - This looks great; the tutorial is much smoother! The only thing is that it looks like the font-weight for the tutorial content and NEXT were both changed to bold when only the NEXT text should have changed

BEFORE | AFTER | DESIGN
-- | -- | --
![image](https://user-images.githubusercontent.com/41650164/92609346-162aed80-f27c-11ea-92f4-53db6278398d.png) | ![image](https://user-images.githubusercontent.com/41650164/92609383-1f1bbf00-f27c-11ea-8762-708d5440dd90.png) | ![image](https://user-images.githubusercontent.com/41650164/92609419-293dbd80-f27c-11ea-8887-34c2f608e8b6.png)

## [Design Specs](https://app.zeplin.io/project/5e3b2d056723900961109141/screen/5e3b335bf14152ce07dc7e9c)
**Tutorial Paragraph Text**
![image](https://user-images.githubusercontent.com/41650164/92609574-58542f00-f27c-11ea-805c-0448442ab324.png)

**Progress (NEXT) Text**
![image](https://user-images.githubusercontent.com/41650164/92609599-60ac6a00-f27c-11ea-8359-c8900028c146.png)

**Comment by milindmore22** [comment]
@ksmoore125 
adding `font-weight:500` will apply medium fonts
so I will be adding `font-weight: normal` for descriptions and skip button, which will match the design.

Code snippet of fonts being applied
```css
@font-face {
	font-family: HelveticaNeue;
	src: url(../fonts/HelveticaNeue.ttf);
	font-weight: normal;
	font-style: normal;
	font-stretch: normal;
}

@font-face {
	font-family: HelveticaNeue;
	src: url(../fonts/HelveticaNeue-Medium.ttf);
	font-weight: 500;
	font-style: normal;
	font-stretch: normal;
}

@font-face {
	font-family: HelveticaNeue;
	src: url(../fonts/HelveticaNeue-Bold.ttf);
	font-weight: bold;
	font-style: normal;
	font-stretch: normal;
}
```

**Comment by ksmoore125** [comment]
I am not quite sure what happened but some of the sites styles appear to be messed up, even in an incognito window.
![image](https://user-images.githubusercontent.com/41650164/92751290-73db3a80-f34d-11ea-8e2a-2a86b1301ada.png)

**Comment by tomasstark** [comment]
@ksmoore125 Ooops, that might be me related to some style updates I did in [CLIENTS-687](https://laterpay.atlassian.net/browse/CLIENTS-687). Will patch those up tomorrow.

**Comment by tomasstark** [comment]
Those changes have been addressed so this can be retested now.

**Comment by tomasstark** [comment]
This is released in [1.2.0](https://github.com/laterpay/revenue-generator/releases/tag/1.2.0), closing the issue now.
